pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -302,7 +302,7 @@ def on_validation_epoch_end(self, *_: Any) -> None:
         _update_n(self.val_progress_bar, self._val_processed)
 
     def on_validation_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
-        if self._main_progress_bar is not None and trainer.state.fn == "fit":
+        if self._main_progress_bar is not None and trainer.state.fn == "fit" and not trainer.sanity_checking:
```

Comment:
removing computation for sanity check since for that we use an invisible main_progress_bar that does not display any logs.

Reply:
I think we should keep it because that's the point of sanity checking, trying the code before even though it does not display anything valuable